proto/storage/process_listening_on_port.proto

```diff
@@ -57,4 +58,6 @@ message ProcessListeningOnPortStorage {
     // ProcessIndicator will be not empty only for those cases when we were not
     // able to find references process in the database
     ProcessIndicatorUniqueKey process = 7;
+
+    string deployment_id = 8 [(gogoproto.moretags) = 'search:"Cluster ID,store"'];
```

Comment:
deployment_id is needed here as we now need the plop record to have the deployment_id.

Reply:
I guess the name should be `Deployment ID`, fortunately there is a such search category.

Reply:
Done, but I am not 100% that the change I made is correct.